### PR TITLE
Universal/bugfix/case 1250602

### PR DESCRIPTION
**Pull request by eh-unity**
### Checklist for PR maker
- [x] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [x] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [x] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
Fix blackscreen rendering bug on OpenGL.

backport 8.x.x: https://github.com/Unity-Technologies/Graphics/pull/942
backport 7.x.x: https://github.com/Unity-Technologies/Graphics/pull/943
backport 9.x.x: https://github.com/Unity-Technologies/Graphics/pull/946
---
### Testing status

**Manual Tests**: What did you do?
- [x] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in developer mode, you have a button at end of resources that check the paths)
- Other: 

Manually tested in editor that this does fix the issue. Added a testcase for it and tested that it shows similar results.

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)
https://github.com/Unity-Technologies/Graphics/blob/fddea223acfec9e9e8c7a4d781ba7afe9748219c/TestProjects/UniversalGraphicsTest/Assets/ReferenceImages/Linear/OSXEditor/Metal/None/111_CameraStackMSAA.png

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.universal/CHANGELOG.md [hunk 1]
```diff
@@ -199,6 +199,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed an issue where preset button could still be used, when it is not supposed to. [case 1246261](https://issuetracker.unity3d.com/issues/urp-reset-functionality-does-not-work-for-renderobject-preset-asset)
 - Fixed issue with Model Importer materials using the Legacy standard shader instead of URP's Lit shader when import happens at Editor startup.
 - Fixed an issue where only unique names of cameras could be added to the camera stack.
+- Fixed an issue where camera stacking with msaa on OpenGL resulted a black screen [case 1250602](https://issuetracker.unity3d.com/issues/urp-camera-stacking-results-in-black-screen-when-msaa-and-opengl-graphics-api-are-used)
```
Comment (pbbastian) [inline com.unity.render-pipelines.universal/CHANGELOG.md:202]:
```suggestion
- Fixed an issue where camera stacking with MSAA on OpenGL resulted in a black screen. [case 1250602](https://issuetracker.unity3d.com/issues/urp-camera-stacking-results-in-black-screen-when-msaa-and-opengl-graphics-api-are-used)
```

**Comment by pbbastian** [comment]
@eh-unity What's the CI status on this?

**Comment by eh-unity** [comment]
This is a oneline change to URP. I would think anything outside of URP is instability.
The metal editor seems to render with incorrect fov. I did adjust the fov for the camera. I would imagine all platforms would fail if it's something I forgot, but if it's metal only then we probably uncovered another bug. 
I think I tested locally with metal when making this and it worked then.
It needs some investigation.

**Comment by eh-unity** [comment]
I pulled the PR with the merged master. I tested locally. The metal does seem to work, both playmode and standalone.

**Comment by eh-unity** [comment]
I added a filter for the test case on MacOSX where it fails. Merged master as it had some changes to the filter as well.

**Pull request merged**